grapple/types/streamfield.py

```diff
@@ -381,12 +381,22 @@ class Meta:
 
         def resolve_image(self, info, **kwargs):
             return self.value
+        
+    class ImageBlock(graphene.ObjectType):
+        image = graphene.Field(get_image_type, required=False)
```

Comment:
to check: whether we need to surface alt_text and whether this is decorative or not